blocks/fourier/include/gnuradio-4.0/fourier/fft.hpp

```diff
@@ -228,5 +228,6 @@ On the choice of window (mathematically aka. apodisation) functions
 
 ENABLE_REFLECTION_FOR_TEMPLATE_FULL((typename T, typename U, template<typename, typename> typename FourierAlgoImpl), (gr::blocks::fft::FFT<T, U, FourierAlgoImpl>), //
                                     in, out, algorithm, fftSize, window, outputInDb, outputInDeg, unwrapPhase, sample_rate, signal_name, signal_unit, signal_min, signal_max);
+// NOTE: FFT can not be registered to the block registry (GR_REGISTER_BLOCK) as it has a template template parameter
```

Comment:
As described above, we need to make this use-case work ... otherwise we fall back to either external code-generation or spelling-out all type/NTTP combinations.